react/Installments.tsx

```diff
@@ -22,42 +37,30 @@ const Installments: StorefrontFC<BasicPriceProps> = props => {
     return null
   }
 
-  let maxInstallments: ProductTypes.Installment | undefined
+  let [maxInstallment] = commercialOffer.Installments
```

Comment:
I think changing this from `maxInstallments` to `maxInstallment` could be misleading. The max _installments_, would be the payment options with the highest number of individual installments. While the max _installment_ would be the highest individual installment value, which in this case is the opposite of what you want, right?